src/Pages/Profile/MemberView/InfoCard.js

```diff
@@ -53,7 +53,8 @@ export default function ProfileCard (props) {
       <div id='profile-box'>
         {props.fields.map((field, ind) => (
           <h3 key={ind} id='inner-text-top'>
-            <b>{field.title}:</b> {field.value}
+            <b>{field.title}:</b>{' '}
+            <nobr dangerouslySetInnerHTML={{ __html: field.value }} />
```

Comment:
why are you doing this line? i dont like injecting values into HTML. it leads to [xss](https://www.youtube.com/watch?v=L5l9lSnNMxg)

Reply:
```html
<b>{field.title}:</b>{' '}
<span className={field.something ? 'red-text' : ''}>{field.value}</span>
```